src/Assetic/Filter/CoffeeScriptFilter.php

```diff
@@ -51,22 +52,22 @@ public function filterLoad(AssetInterface $asset)
         $input = FilesystemUtils::createTemporaryFile('coffee');
         file_put_contents($input, $asset->getContent());
 
-        $pb = $this->createProcessBuilder($this->nodeBin
+        $commandline =$this->nodeBin
             ? array($this->nodeBin, $this->coffeeBin)
-            : array($this->coffeeBin));
+            : array($this->coffeeBin);
 
-        $pb->add('-cp');
+        array_push($commandline, '-cp');
```

Comment:
using `$commandline[] = '-cp';` would be more readable than lots of `array_push` IMO

Reply:
My motivation here is to make it work with Symfony v4. There are certainly a lot of ways one could improve this.